docs/changelog/1_0_b1.rst

```diff
@@ -0,0 +1,332 @@
+==========
+1.0 Beta 1
+==========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 beta 1 "Sirius".
```

Comment:
Maybe we should call this Sirius A. Then the second beta or the RC can be called Sirius B. Thoughts?

Reply:
That would be kinda neat, but only if we pretty much nail down the guarantee that there will be exactly 2 betas, no more no less. With more betas the naming wouldn't really be as clean then (and with only 1 beta it would be odd, too).